.github/workflows/lint.yml

```diff
@@ -0,0 +1,36 @@
+name: lint
+on:
+  push:
+    branches:
+      - master
+      - dev
+    tags:
+      - v*
+  pull_request:
+    branches:
+      - dev
+      - master
+
+jobs:
+  test:
+    runs-on: "ubuntu-latest"
+    timeout-minutes: 10
+    strategy:
+      matrix:
+        node-version: [14.x]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+
+      - name: Install dependencies
+        run: npm install
+
+      # This file is required by most controllers and triggers "import/no-unresolved" if missing
+      - name: Copy config file
+        run: cp ./__tests__/integration/configuration/config-fixture.js ./src/config.js
+
+      - name: Lint
+        run: npm run lint
```

Comment:
Since our tests are currently not green, I tried to create the minimum possible interference with them. So I created a new workflow _yml_.

But now that you mention, I see our [wallet mobile](https://github.com/HathorNetwork/hathor-wallet-mobile/blob/master/.github/workflows/main.yml) follows this practice of running the lint within the `main` workflow.

I agree that's a good idea, but I would rather make all tests green before making this move.
What do you think?

Reply:
I think the interference caused by linters is not that big. We will be able to differentiate failures caused by them or by the tests.

Reply:
I tried adding the linter to the main workflow on [a555448](https://github.com/HathorNetwork/hathor-wallet-headless/pull/174/commits/a555448eef4c6e42f73094ff903b1ba461e5ec83), but it broke when tried to run on Node 8.x, possibly also on 10.x

We could use additional actions or configurations to enable the linting only on specific node versions, but I think there is little benefit for this extra complexity.

Do you have any suggestions on this?